lib/worker.js

```diff
@@ -2,10 +2,45 @@
 
 const net = require('net');
 const JSON = require('./json-buffer');
+const fs = require('fs');
+const os = require('os');
+const path = require("path");
+const READFILETIMELIMIT = 30000;
+const debugMode = false;
+
+const responseFolder = path.join(os.tmpdir(), 'sync_rpc_responses');
+if (!fs.existsSync(responseFolder)) {
+  fs.mkdirSync(responseFolder, {recursive: true});
+}
+
+const responseFile = path.join(responseFolder, `responseFile.txt`);
```

Comment:
I think we can share this data when creating worker thread using workerData param from the parent thread so that we will have single source of truth.